package.json

```diff
@@ -22,40 +22,41 @@
     "tslib": "^1.10.0"
   },
   "peerDependencies": {
-    "react": "^16.11.0"
```

Comment:
I don't think this should be updated - resub works just fine with react 16.11, there's no real dep on 16.12 & peerDeps are managed by the consumer